trillian/migrillian/configpb/config.proto

```diff
@@ -83,6 +83,14 @@ message MigrationConfig {
   // The function that computes LeafIdentityHash for Trillian log entries.
   IdentityFunction identity_function = 12;
 
+  // This flag is used in continuous mode to work around logs that don't expose
```

Comment:
I'd reword this comment a bit as (while somewhat ambiguous) the RFC does not define the get-entry-and-proof entrypoint as optional. Arguably these logs are not complying with the standard. Maybe just remove the second half of the first sentence.

Reply:
Extended the RFC comment a bit. How does it look now?

Reply:
I'd still remove the "expose" bit and and just say that with the option on it doesn't require get-entry-and-proof.

Reply:
Done rewording. Could you suggest a specific wording if this one still doesn't look ok? Not sure how to properly cut the corners here.